WordPress/src/main/java/org/wordpress/android/widgets/FlowLayout.java

```diff
@@ -1,144 +0,0 @@
-/*
- * Sample FlowLayout wrote by Romain Guy: http://www.parleys.com/play/514892280364bc17fc56c0e2/chapter38/about
- * Fixed and tweaked since
- */
-
-package org.wordpress.android.widgets;
-
-import android.content.Context;
-import android.content.res.TypedArray;
-import android.util.AttributeSet;
-import android.view.View;
-import android.view.ViewGroup;
-
-import androidx.core.view.ViewCompat;
-
-import org.wordpress.android.R;
-
-public class FlowLayout extends ViewGroup {
-    private int mHorizontalSpacing;
-    private int mVerticalSpacing;
-
-    public FlowLayout(Context context, AttributeSet attrs) {
-        super(context, attrs);
-
-
-        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.FlowLayout);
-        try {
-            mHorizontalSpacing = a.getDimensionPixelSize(R.styleable.FlowLayout_horizontalSpacing, 0);
-            mVerticalSpacing = a.getDimensionPixelSize(R.styleable.FlowLayout_verticalSpacing, 0);
```

Comment:
We can also remove `FlowLayout` and `FlowLayout_Layout` in `attrs.xml`

Reply:
Thanks @irfano for pointing it out. Removed both in [fcc6fc3](https://github.com/wordpress-mobile/WordPress-Android/pull/19788/commits/fcc6fc3b140fced5c2188cc1ad2cb7e3ff473979)